Content.Server/Weapons/Melee/Shield/EnergyShieldComponent.cs

```diff
@@ -0,0 +1,33 @@
+using Robust.Shared.Audio;
+
+namespace Content.Server.Weapons.Melee.EnergyShield;
+
+[RegisterComponent]
```

Comment:
```suggestion
[RegisterComponent, NetworkedComponent]
```

When it's moved into shared.

Reply:
done